### PR TITLE
Reach - Removes circuit imprinter from wall

### LINKED ISSUE #31253: Reach: Circuit imprinter mapped into a wall
Reported on Discord by SpaceLizard
grafik

### PR DESCRIPTION
**Pull request by JustCone14**
## About the PR

Maybe a misclick from the last reach update, there was a circuit imprinter mapped into the shuttle wall. this PR removes it.

## Why / Balance

fixes #31253

### CODE REVIEW

File: Resources/Maps/reach.yml [hunk 1]
```diff
@@ -11645,13 +11642,6 @@ entities:
     - type: Transform
       pos: 8.5,-8.5
       parent: 2
-- proto: MaterialReclaimer
-  entities:
-  - uid: 1662
-    components:
-    - type: Transform
-      pos: 10.5,-8.5
-      parent: 2
```
Comment (Brandon-Huu) [inline Resources/Maps/reach.yml:11654]: This isn't a circuit imprinter

Comment (Brandon-Huu) [reply Resources/Maps/reach.yml:11654]: Guess it's from this PR
https://github.com/space-wizards/space-station-14/pull/30802

Comment (JustCone14) [reply Resources/Maps/reach.yml:11654]: yeah I didn't remove that. I guess it might have been removed automatically because of the recycler PR

**Review by Brandon-Huu — APPROVED** [review]
image

**Pull request merged**